src/AeroSharp/AeroSharp.csproj

```diff
@@ -1,47 +1,42 @@
 <Project Sdk="Microsoft.NET.Sdk">
-
-  <PropertyGroup>
-    <TargetFramework>netstandard2.1</TargetFramework>
-    <GeneratePackageOnBuild>true</GeneratePackageOnBuild>
-    <Version>0.3.1</Version>
-    <Authors>Wayfair</Authors>
-    <Description>Wrapper around the .NET Aerospike client that provides a variety of methods for storing and retrieving data in Aerospike</Description>
-    <Copyright>Wayfair ©2021</Copyright>
-    <PackageLicenseFile>LICENSE</PackageLicenseFile>
-    <PackageProjectUrl>https://github.com/wayfair-incubator/AeroSharp</PackageProjectUrl>
-    <RepositoryUrl>https://github.com/wayfair-incubator/AeroSharp</RepositoryUrl>
-    <RepositoryType>git</RepositoryType>
-    <PackageTags>aerospike, aerosharp, serialization, policy validation</PackageTags>
-    <PackageReadmeFile>README.md</PackageReadmeFile>
-  </PropertyGroup>
-
-  <ItemGroup>
-    <None Include="..\..\README.md">
-      <Pack>True</Pack>
-      <PackagePath>\</PackagePath>
-    </None>
-    <None Include="..\..\LICENSE">
-      <Pack>True</Pack>
-      <PackagePath></PackagePath>
-    </None>
-    <None Remove="stylecop.json" />
-  </ItemGroup>
-
-  <ItemGroup>
-    <AdditionalFiles Include="..\..\stylecop.json" Link="stylecop.json" />
-  </ItemGroup>
-
-  <ItemGroup>
-    <PackageReference Include="Aerospike.Client" Version="5.1.1" />
-    <PackageReference Include="FluentValidation" Version="11.0.3" />
-    <PackageReference Include="lz4net" Version="1.0.15.93" />
-    <PackageReference Include="MessagePack" Version="2.3.112" />
-    <PackageReference Include="Polly" Version="7.2.3" />
-    <PackageReference Include="protobuf-net" Version="3.1.4" />
-    <PackageReference Include="StyleCop.Analyzers" Version="1.1.118">
-      <PrivateAssets>all</PrivateAssets>
-      <IncludeAssets>runtime; build; native; contentfiles; analyzers; buildtransitive</IncludeAssets>
-    </PackageReference>
-  </ItemGroup>
- 
-</Project>
+	<PropertyGroup>
+		<TargetFramework>net6.0</TargetFramework>
```

Comment:
Really the only change here - the rest I _believe_ is whitespace only.  I think my Mac got a little fancy with formatting.

Reply:
Looking at the changes here, it unfortunately looks like we have double tabs rather than single? It would be great if we could revert that to keep the original formatting.

Reply:
What are your thoughts on supporting multiple target frameworks? We could also start supporting net7.0.

Reply:
Hmm - that could set us up for the future!  I like the idea and the amount improvements in 7 is awesome (according to Toub: https://devblogs.microsoft.com/dotnet/performance_improvements_in_net_7/) 

Let me try that out and see if it breaks things a whole bunch.

Reply:
Agreed on the formatting - lemme fix that up.